mitodl/micromasters
# Added logging and email config

#### What are the relevant tickets?
Closes #32 
Closes #35 

#### What's this PR do?
Adds logging and email configuration settings, as well as app.json

#### Where should the reviewer start?
This is essentially copied from TeachersPortal configuration, so it should only require light review. A good place to start is to check that the application emits logs when `DEBUG` is set to `False`.
Also, that there are no "MICROMASTERS" spelling errors from when I typed it out a couple of times.

#### How should this be manually tested?
In addition to the above, one could set one of the valid `MICROMASTERS_...` vars to true in the docker-compose environment (docker-compose config) and verify that the corresponding setting was applied in the django settings.

#### Any background context you want to provide?
This is one of those things I need to get around to adding to the cookiecutter.

#### What GIF best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/QqkRs73FlKO52/giphy.gif)
